lumen_runtime/src/otp/erlang/date_0/test.rs

```diff
@@ -0,0 +1,10 @@
+use crate::otp::erlang::date_0::native;
+use crate::scheduler::with_process;
+
+#[test]
+fn returns_date_as_a_three_tuple() {
+    with_process(|process| {
```

Comment:
This should check that it is actually a 3 Tuple of integers.  The test description does not match the asserts.